src/python/pants/help/help_info_extracter_test.py

```diff
@@ -96,6 +97,14 @@ def test_deprecated(self):
     self.assertEqual('do not use this', ohi.removal_hint)
     self.assertIsNotNone(ohi.deprecated_message)
 
+  def test_enum(self):
```

Comment:
It's a good practice to add `-> None` to test functions. Why? Because then MyPy will check the implementation of the test, whereas it would normally skip it.

```suggestion
  def test_enum(self) -> None:
```

(No worries about adding it to other functions in this file - want to avoid scope creep.)

Reply:
Good call, I need to commit typing habits to heart 😃